pkg/routeagent_driver/handlers/ovn/bridge.go

```diff
@@ -0,0 +1,90 @@
+/*
+© 2021 Red Hat, Inc. and others
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ovn
+
+import (
+	"net"
+	"syscall"
+
+	"github.com/pkg/errors"
+	"github.com/vishvananda/netlink"
+
+	"github.com/submariner-io/submariner/pkg/routeagent_driver/handlers/ovn/vsctl"
+)
+
+func (ovn *Handler) getMTU() (int, error) {
+	link, err := ovn.netlink.LinkByName(OVNK8sMgmntIntfName)
+
+	if err != nil {
+		if _, ok := err.(netlink.LinkNotFoundError); !ok {
+			return 0, errors.Wrapf(err, "error retrieving link by name %q", OVNK8sMgmntIntfName)
+		}
+	}
+
+	return link.Attrs().MTU, nil
```

Comment:
If LinkNotFound, this would give an error.

Reply:
Oh right, thanks.

Reply:
Done.